libs/payments/capability/.swcrc

```diff
@@ -0,0 +1,30 @@
+{
+  "jsc": {
+    "target": "es2017",
+    "parser": {
+      "syntax": "typescript",
+      "decorators": true,
+      "dynamicImport": true
+    },
+    "transform": {
+      "decoratorMetadata": true,
+      "legacyDecorator": true
+    },
+    "keepClassNames": true,
+    "externalHelpers": true,
+    "loose": true
```

Comment:
Why enable this? The docs are rather rough

Reply:
Ah, this was part of the Nx default SWC generator output. I've trimmed the .swcrc a bit to mostly only include what's needed for unit tests, so this line has been removed.